sdk/metric/instrument.go

```diff
@@ -202,6 +204,10 @@ func (i *int64Inst) Record(ctx context.Context, val int64, opts ...metric.Record
 	i.aggregate(ctx, val, c.Attributes())
 }
 
+func (i *int64Inst) Enabled(_ context.Context) bool {
```

Comment:
This is a still an addition to the public, stable artifacts of the metrics SDK, right?  I thought we had concluded that we would use feature branches for experimental API features in https://github.com/open-telemetry/opentelemetry-go/issues/5882?

Reply:
The types are not exported. Therefore I think it is an acceptable addition.

Reply:
ah, right. Thanks